tests/Storage/relation/migrations/1970_01_01_000002_constraint_task_tag_relation.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ConstraintTaskTagRelation extends Migration
```

Comment:
__[PHP_CodeSniffer]__ Each class must be in a namespace of at least one level (a top-level vendor name) ([link](https://sider.review/gh/repos/151422991/pulls/73#issue-4648830))

<sub>posted by Sider</sub>